api/object-handlers.go

```diff
@@ -1246,6 +1262,11 @@ func (api ObjectAPIHandlers) NewMultipartUploadHandler(w http.ResponseWriter, r
 			WriteErrorResponse(w, r, err)
 			return
 		}
+	} else if configuration, ok := checkBucketEncryption(bucketName, api.ObjectAPI); ok {
```

Comment:
Could get bucketencryption by bucket info？

Reply:
I think some request have not such detailed bucket info